app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt

```diff
@@ -0,0 +1,191 @@
+package com.bridgewell.quickstart.android.activity
+
+import android.os.Bundle
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.ImageButton
+import android.widget.ImageView
+import android.widget.LinearLayout
+import androidx.appcompat.app.AppCompatActivity
+import androidx.viewpager2.widget.ViewPager2
+import com.bridgewell.bwmobile.ads.inapp.InAppApi
+import com.bridgewell.bwmobile.ads.inapp.listener.BwsAdViewListener
+import com.bridgewell.bwmobile.ads.inapp.model.BwsAdView
+import com.bridgewell.quickstart.android.R
+import com.bridgewell.quickstart.android.activity.ui.adapter.TabbedViewAdapter
+import com.bridgewell.quickstart.android.data.AdType
+import com.bridgewell.quickstart.android.utils.showToast
+import com.google.android.material.tabs.TabLayout
+import com.google.android.material.tabs.TabLayoutMediator
+import koleton.api.hideSkeleton
+import koleton.api.loadSkeleton
```

Comment:
![image](https://github.com/user-attachments/assets/1ec0895f-7e30-4b56-8840-aead227e6ba8)

Is this being used?

Reply:
Yes it's being used by the loading animation of the skeleton